utils/storage/cookie.js

```diff
@@ -75,6 +69,20 @@ class CookieLocal {
       return false;
     }
   }
+
+  /**
+   * Function to check cookie support exists or not
+   * @returns boolean
+   */
+  IsCookieSupported() {
+    this.set("rudder_cookies", true);
+
+    if (this.get("rudder_cookies")) {
+      this.remove("rudder_cookies");
+      return true;
+    }
+    return false;
```

Comment:
Two things:
- Rename it to `test_rudder_cookie`
- Reuse the cookie name rather than repeat it in multiple places.

Reply:
Addressed